packages/backend-core/src/db/errors.ts

```diff
@@ -0,0 +1,9 @@
+export function is409Error(error: any) {
```

Comment:
NAB: Might be worth making this generic, allowing an error code to be passed as the second param.